sys/include/net/ng_pktdump.h

```diff
@@ -29,18 +29,41 @@ extern "C" {
 #endif
 
 /**
- * @brief   Start the packet dump thread and listening for incoming packets
+ * @brief   Message queue size for the pktdump thread
+ */
+#ifndef NG_PKTDUMP_MSG_QUEUE_SIZE
+#define NG_PKTDUMP_MSG_QUEUE_SIZE       (8U)
+#endif
+
+/**
+ * @brief   Priority of the pktdump thread
+ */
+#ifndef NG_PKTDUMP_PRIO
+#define NG_PKTDUMP_PRIO                 (PRIORITY_MAIN - 1)
```

Comment:
Something higher than `MAIN` it's not that important, that the output comes as fast as possible.